### DISCUSSION TITLE
explain it more with example

### DISCUSSION THREAD

**Discussion by VenomFate-619** [jdevfullstack-tutorials/resolution-and-screen-size#1] [Q&A]
**This will be truly pixelated for normal viewing distance for a computer monitor.** 
Can you explain it more with visuals, I am not getting it.

**Comment by jfullstackdev** [comment]
If you experience using a Smart TV 1080p, say, 44", if you are viewing a video from far enough, that is not an issue. But when you sit down and you use that as a computer monitor, you tend to sit close to the monitor. You will be seeing much pixels even for 27" 1080p. 

The same is also true for a 21.5" 4k monitor just like the iMac that I'm using, issued by my company. When I'm viewing a video from far enough, you can't tell the difference of it from a 1080p. But again, you will see the sense of it when you sit down to use it as a computer monitor. Basically, the starting point is when you stretch your arm, either you will be closer or farther from it depending on your preference and/or your visual acuity. At this point, you will appreciate that 4k resolution for a 21.5" screen for productivity because you don't see any individual pixel or Apple calls it as `retina`.

I wrote this because there are several ambiguous statements. But Apple have the best way of implementing this.
